### ISSUE TITLE
Jailbreak bypass

### ISSUE THREAD

**Issue by the1ulike** [pwn20wndstuff/Undecimus#2039]
I have simple app called QueQ but can not go through bypass
Any advise? Thank you

**Comment by SarKaa** [comment]
This is nothing to do with the jailbreak itself, ask on reddit or something, or maybe try google, because the jailbreak devs aren't gonna do anything about it.
Your question is pointless, you simply cannot ask a dev about bypasses, thats a problem you have to accept when you jailbreak, and if you want a bypass, you should ask other devs that already make bypasses, such as kernbypass (not recommended) or liberty lite

**Comment by the1ulike** [comment]
If my question is pointless, why you respond?! your unrecommended and recommended bypasses still not work. Try and see
The only way to work for me is to get an old version of this app ( I used AppStore++ to get an old version).. still the app (with or without bypasses) won't work until back to unjailbreak stage (by a reboot). but still, this is inconvenient and looking for a better way

**Comment by SarKaa** [comment]
I responded because I'm still trying to help, but you shouldn't be asking for help here, this is completely unrelated to the actual jailbreak, you should make a post on reddit,
The issues here are for actual important and relevant problems, jailbreak detection has been a problem for years, and if you are part of the jailbreak community you should know about the problems. 
I am simply trying to help the dev by clearing up some useless issues, which he shouldn't have to deal with. And why would you ask why I responded, do you want to be helped or not?
In short, ask on reddit, and close this issue, and stop filing useless questions like this

**Comment by the1ulike** [comment]
bypassing jailbreak not related to jailbreak?! ok thank you

**Comment by SarKaa** [comment]
No that’s not what I’m saying, you are essentially blaming the jailbreak dev, ie pwn20wnd that the app you need has jailbreak detection, when you should be asking for help on reddit. 
Jailbreak detection is not relevant here in an unc0ver issue report

**Comment by the1ulike** [comment]
blaming?! check again what I have said in the first line
I can see you keep promoting Reddit but unc0ver needs to know the feedback and all issues need to be discussed

**Comment by SarKaa** [comment]
But unc0ver and pretty much every single jailbreak out there knows about this, this is basic stuff, every single person that jailbreaks knows that some apps can’t be used, and what makes pwn20wnd different. This is completely irrelevant and shouldn’t be here end of

**Comment by SarKaa** [comment]
Your not doing the jailbreak community a favour by asking here, in fact, your just making things harder for the devs,

**Comment by the1ulike** [comment]
Nothing easy. Developers looking for challenges 
unc0ver 5.0.1 better than the previous versions but we need to report apps that can not bypass the jailbreak to let devs do a better job
I have reported a simple app that should be easy to bypass
apps like "QNB Mobile" is more challenging

**Comment by SarKaa** [comment]
I know what you are trying to say, but this ain’t the place, pwn20wnd has nothing to do with any jailbreak bypasses, and I doubt he would want to try, If you had to report an app that doesn’t work this isn’t the place to do it. I bet you that the app doesn’t work with checkra1n either, which means that it isn’t anything to do with the actual jailbreak, which means this isn’t the place you should be complaining. Instead of arguing you could help yourself by asking in discord or reddit, which is not advertising, but they are the 2 places someone which you are supposed to go to, not the unc0ver issues
